opentracing-api/src/main/java/io/opentracing/tag/ShortTag.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.tag;
+
+public class ShortTag extends AbstractTag<Short> {
```

Comment:
per http://opentracing.io/spec/#the-span-interface, I would have expected NumericTag (rather than IntTag and ShortTag), referencing https://docs.oracle.com/javase/7/docs/api/java/lang/Number.html ... And BooleanTag is missing from this PR, I think.